dashboard/app/controllers/api/v1/assessments_controller.rb

```diff
@@ -1,8 +1,14 @@
 class Api::V1::AssessmentsController < Api::V1::JsonApiController
   include LevelsHelper
-  load_and_authorize_resource :section
+
+  before_filter :load_from_cache
```

Comment:
FYI `before_filter` is deprecated in Rails 5. In the future, use `before_action` instead. 

This is logging deprecation warnings in the Cronjobs, causing Honeybadger errors (we're pretty aggressive with capturing anything logged to stderr in our cronjobs). 

Fix: https://github.com/code-dot-org/code-dot-org/pull/23176